iocore/net/OCSPStapling.cc

```diff
@@ -308,10 +315,10 @@ class HTTPRequest : public Continuation
     sin.sin_port        = 65535;
 
     this->_fsm = FetchSMAllocator.alloc();
-    if (use_post) {
-      this->_fsm->ext_init(this, "POST", uri, "HTTP/1.1", reinterpret_cast<sockaddr *>(&sin), TS_FETCH_FLAGS_SKIP_REMAP);
-    } else {
+    if (use_get) {
       this->_fsm->ext_init(this, "GET", uri, "HTTP/1.1", reinterpret_cast<sockaddr *>(&sin), TS_FETCH_FLAGS_SKIP_REMAP);
+    } else {
+      this->_fsm->ext_init(this, "POST", uri, "HTTP/1.1", reinterpret_cast<sockaddr *>(&sin), TS_FETCH_FLAGS_SKIP_REMAP);
     }
```

Comment:
I suggest setting the char* in a conditional:

```cpp
    char const *const method = use_get ? "GET": "POST";
    this->_fsm->ext_init(this, method, uri, "HTTP/1.1", reinterpret_cast<sockaddr *>(&sin), TS_FETCH_FLAGS_SKIP_REMAP);

```

Reply:
Embedded the ternary operator into the ext_init line.